File: Baekjoon_Online_Judge/NO_13752/13752.cpp
#include <iostream>
using namespace std;
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 08                 | 
 | Day 02                   | 
 | 13752 히스토그램           | 
 ---------------------------- 
*/

int main(){
    ios_base::sync_with_stdio(0);
    cin.tie(0);
    cout.tie(0);
    
    int n;

    cin >> n;

    for(int i = 0; i < n; i ++){
        int k;

        cin >> k;

        for(int j = 0; j < k; j ++) cout << "=";

        cout << "\n";
    }
    return 0;
}

File: Baekjoon_Online_Judge/NO_13752/13752.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 08                 | #
# | Day 02                   | #
# | 13752 히스토그램           | #
# ---------------------------- #


if __name__ == "__main__":
    n = int(r_input())

    for _ in range(n):
        k = int(r_input())

        print('=' * k)
